R/pkg/R/types.R

```diff
@@ -47,6 +47,19 @@ COMPLEX_TYPES <- list(
 # The full list of data types.
 DATA_TYPES <- as.environment(c(as.list(PRIMITIVE_TYPES), COMPLEX_TYPES))
 
+SHORT_TYPES <- as.environment(list(
+  "character"="chr",
```

Comment:
Could you match the format / spacing as above:
https://github.com/apache/spark/pull/9613/files#diff-98ce596228de621ff95b157a80d7f953R31